build/build.ps1

```diff
@@ -0,0 +1,155 @@
+<#
+    .SYNOPSIS
+    Build a Docker image for Business Central.
+
+    .DESCRIPTION
+    This script builds a Docker image for Business Central using the specified parameters.
+    It also pushes the image to a specified Azure Container Registry if requested.
+
+    .PARAMETER BaseImage
+    The base image to use for building the Docker image (mcr.microsoft.com/dotnet/framework/runtime or mcr.microsoft.com/windows/servercore)
+    Example: mcr.microsoft.com/dotnet/framework/runtime:4.8.1-windowsservercore-ltsc2025
+
+    .PARAMETER LtscTag
+    The LTSC tag to use for the image. 
+    For example: ltsc2016, ltsc2019, ltsc2022 or ltsc2025
+
+    .PARAMETER FilesOnly
+    A switch indicating whether to build only the files.
+
+    .PARAMETER Only24
+    A switch indicating whether to build the BC24 image.
```

Comment:
This switch will cause it to _only_ build the BC24 image?

Reply:
Yeah it's some special image we produce for BC24. Not sure what the story is there. Not something I intend on removing in this PR though :)